CorsixTH/Lua/hospitals/ai_hospital.lua

```diff
@@ -46,6 +46,10 @@ function AIHospital:logTransaction()
   -- AI doesn't need a log of transactions, as it is only used for UI purposes
 end
 
+function AIHospital:hasStaffedDesk()
```

Comment:
Possible future todo?:
Adjust this to only be true when the AI Hospital opens

Reply:
Indeed this will switch with the AI hospitals change to actually detect a receptionist has been hired and a desk exists again. This just gets it working before they are implemented